mitodl/open-discussions
# Make learning path items drag resortable

#### Pre-Flight checklist


#### What are the relevant tickets?
#3912 

#### What's this PR do?
This PR makes learning paths drag re-sortable at `infinite/lists/<learning path id>`.

#### How should this be manually tested?
1. Visit http://od.odl.local:8063/infinite/lists
2. Create and populate some userlists / learning paths if you haven't already. *Populate them via the bookmark icon on search results page, http://.od.odl.local:8063/infinite/search*. Make sure you have at least one learning path and at least one userlist.
3. From http://od.odl.local:8063/infinite/lists, visit the details page for a userlist *not a learning path*. It should not have a "Reorder" button.
4. Now visit a learning path page. It should have a "Reorder" button. Try reordering things:
     - drag items up/down within the list ... the order should be correct after you drag, and also after you reload the page.
     - If you like, try reproducing bug #3944. It should be repredocable on `/learn` but not on `/infinite`. 
5. Visit a public learning path as an anonymous user in private browser window. You should not be able to Reorder.

#### Screenshots (if appropriate)

**Desktop:**

https://user-images.githubusercontent.com/9010790/233116127-74a3f4b2-16ef-4aa1-be89-ab396f415820.mov

**Mobile:**
<img width="462" alt="Screenshot 2023-04-19 at 11 04 58 AM" src="https://user-images.githubusercontent.com/9010790/233118446-8e9299c9-3899-4dc9-a463-ff786a9c39eb.png">
